Sources/Caching/LargeItemCacheType.swift

```diff
@@ -46,7 +46,7 @@ extension FileManager: LargeItemCacheType {
 
         // Set up file handling
 
-        let tempFileURL = temporaryDirectory.appendingPathExtension(checksum?.value ?? url.pathExtension)
+        let tempFileURL = temporaryDirectory.appendingPathComponent((checksum?.value ?? "") + url.lastPathComponent)
```

Comment:
This ended up being an issue too after changing the other file, moving didn't work. So I had to update this too.